sketch_blink.ino.ino

```diff
@@ -0,0 +1,44 @@
+
+/*this fade sketch meets the Acceptance Criteria https://trello.com/c/HQigFpAG */
+
+int pin = 9;
+int shine = 0;
+int fadeAmount = 5.9;
+int flashon = 258;
+int flashoff = 250;
+
```

Comment:
And always remember to use a camel case for the flash on and flash off to work and other variable names in your Arduino coding James. #NeverGiveUpJames#